core/cap-0040.md

```diff
@@ -347,7 +347,16 @@ None yet.
 
 ## Security Concerns
 
-None known.
+If a transaction requires the signature of a Signed Payload Signer and a signer
+signs it without understanding the preimage of the payload, the signer could be
+authorizing another Stellar transaction if the payload is a Stellar transaciton
+hash, or the signer could be authorizing some other operation on another network
+or application. People and applications should never sign a transaction they do
+not completely understand. There are many cases where signing a transaction not
+completely understood can pose a security concern for the signer because the
+signer could be authorizing something they would not approve of if they did
+understand the transaction. This proposal introduces a new case with the Signed
+Payload Signer.
```

Comment:
```suggestion
If a transaction requires the signature of a Signed Payload Signer and a signer
signs it without understanding the preimage of the payload, the signer could be
authorizing another Stellar transaction if the payload is a Stellar transaciton
hash, or the signer could be authorizing some other operation on another network
or application. People and applications should never sign a hash that they do
not completely understand the consequences of signing, in all the contexts that
their key provides authentication or authorization. People and applications
should never sign a transaction they do not completely understand. There are
many cases where signing a transaction not completely understood can pose a
security concern for the signer because the signer could be authorizing
something they would not approve of if they did understand the transaction. This
proposal introduces a new case with the Signed Payload Signer.
```